build-docker-image/action.yaml

```diff
@@ -120,7 +124,9 @@ runs:
         npx semantic-release --debug
 
     # use specific commit-sha to be sure this is not changed
-    - uses: easimon/maximize-build-space@b4d02c14493a9653fe7af06cc89ca5298071c66e
+    - name: Maximize build space
+      if: ${{ inputs.maximize-build-space == true }} 
```

Comment:
We can also add the if to https://github.com/neohelden/actions-library/blob/a76e889992549dde194dcceb63404a8b58bf522e/build-docker-image/action.yaml#L130

Reply:
done